src/Surfnet/MessageBirdApiClientBundle/Tests/Service/MessagingServiceTest.php

```diff
@@ -46,14 +48,30 @@ public function testItLogsInvalidMessages()
         $message = new Message('SURFnet', '31612345678', 'body');
         $result = new SendMessageResult(SendMessageResult::STATUS_NOT_SENT, [['code' => SendMessageResult::ERROR_INVALID_PARAMS, 'description' => '']]);
 
-        $libraryService = m::mock('Surfnet\MessageBirdApiClient\Messaging\MessagingService')
-            ->shouldReceive('send')->once()->with($message)->andReturn($result)
+        // Create a mock for Surfnet\MessageBirdApiClient\Messaging\MessagingService
+        $libraryService = $this->getMockBuilder('Surfnet\MessageBirdApiClient\Messaging\MessagingService')
+            ->disableOriginalConstructor()
             ->getMock();
```

Comment:
Is there a specific reason why you utilized the PHPUnit mock builder here instead of Mockery that was originally used?

I'm OK with this, but in a perfect word we would use one or the other, not both. Personally I'm a Mockery fanboy.